prime-react/src/main/scala/react/primereact/PrimeStyles.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.primereact
+
+import react.common._
+
+trait PrimeStyles {
```

Comment:
In observe I'm prefixing our customization classes to prime-react with `pl-` (for "Prime Lucuma"). Maybe we can adopt this or something similar throughout?

Reply:
Actually, our customizations shouldn't go here but rather in lucuma-ui. Are `compact` and `very-compact` part of prime-react or something we are adding?

Reply:
`compact` and `very-compact` are things I added to allow for better spacing. They are used in the layout stuff in lucuma-ui, but I'm also using them to reduce space in buttons, so I left them here. Maybe the best thing to do is to move all of these `customized` facades to lucuma-ui and leave `lucuma-react/primereact` as just the generated facade.

Reply:
> In observe I'm prefixing our customization classes to prime-react with `pl-` (for "Prime Lucuma"). Maybe we can adopt this or something similar throughout?

Sure. Seems reasonable.